weblab_cg/_generator.py

```diff
@@ -186,6 +186,7 @@ def derivative_name(deriv):
     # Create parameter information dicts
     parameter_info = []
     for i, parameter in enumerate(parameters):
+        print(parameter)
```

Comment:
Oops!

Reply:
Not part of this PR. I'll again update it to the latest master, which should see it gone!